Library/Homebrew/build.rb

```diff
@@ -83,7 +83,8 @@ def install
       ENV.keg_only_deps = keg_only_deps
       ENV.deps = formula_deps
       ENV.run_time_deps = run_time_deps
-      ENV.x11 = reqs.any? { |rq| rq.is_a?(X11Requirement) }
+      ENV.x11 = reqs.any? { |rq| rq.is_a?(X11Requirement) } ||
```

Comment:
Should `XorgRequirement` inherit from `X11Requirement` in name only so that `rq.is_a?(X11Requirement)` is true?